mappings/net/minecraft/client/gui/widget/layout/RotatableLayoutWidget.mapping

```diff
@@ -0,0 +1,86 @@
+CLASS net/minecraft/unmapped/C_zuxsitrm net/minecraft/client/gui/widget/layout/RotatableLayoutWidget
+	FIELD f_ochhixwu children Ljava/util/List;
+	FIELD f_wnzzziql orientation Lnet/minecraft/unmapped/C_zuxsitrm$C_ekgfxrpf;
+	FIELD f_xpienxli layoutSettings Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD <init> (IIIILnet/minecraft/unmapped/C_zuxsitrm$C_ekgfxrpf;)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 width
+		ARG 4 height
+	METHOD <init> (IILnet/minecraft/unmapped/C_zuxsitrm$C_ekgfxrpf;)V
+		ARG 1 width
+		ARG 2 height
+		ARG 3 orientation
+	METHOD m_hjuyuxiv (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_zuxsitrm$C_isrpishr;)V
+		ARG 1 child
+	METHOD m_nrxugcux add (Lnet/minecraft/unmapped/C_dpukplka;)Lnet/minecraft/unmapped/C_dpukplka;
+		ARG 1 widget
+	METHOD m_rsgnlyvt add (Lnet/minecraft/unmapped/C_dpukplka;Lnet/minecraft/unmapped/C_pjarmpvz;)Lnet/minecraft/unmapped/C_dpukplka;
+		ARG 1 widget
+		ARG 2 layoutSettings
+	METHOD m_syxvupud getLayoutSettings ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_wauzovfj copyLayoutSettings ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_ywlwqxpm add (Lnet/minecraft/unmapped/C_dpukplka;Ljava/util/function/Consumer;)Lnet/minecraft/unmapped/C_dpukplka;
+		ARG 1 widget
+		ARG 2 layoutSettingsConsumer
+	CLASS C_ekgfxrpf Orientation
+		METHOD m_fnqdcoug setRotatedY (Lnet/minecraft/unmapped/C_zuxsitrm$C_isrpishr;II)V
+			COMMENT Sets the "y" coordinate of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will set the actual y
+			COMMENT coordinate of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL},
+			COMMENT it will set its x coordinate.
+			ARG 1 widget
+			ARG 2 y
+			ARG 3 height
+		METHOD m_jsvnlxcc getRotatedY (Lnet/minecraft/unmapped/C_dpukplka;)I
+			COMMENT Returns the "y" coordinate of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual y
+			COMMENT coordinate of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL},
+			COMMENT it will be its x coordinate.
```

Comment:
```suggestion
			COMMENT Returns the Y coordinate of the given {@code widget} rotated with this orientation.
			COMMENT <p>
			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual Y
			COMMENT coordinate of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL},
			COMMENT it will be its X coordinate.
```

Reply:
Again, there is a reason it is in quotes.  Perhaps it needs to be made clearer if that wasn't clear, how I don't know.